README.md

```diff
@@ -30,6 +30,14 @@ This section describes the terms used for all documentation in this repository.
 
   See [the image builder documentation](image-builder/README.md).
 
+- initrd (or "initramfs")
+
+  A compressed cpio archive loaded into memory and used as part of the Linux
+  startup process. During startup, it is unpacked by the kernel into a special
```

Comment:
Passive voice. Consider changing to:
During startup, the kernel unpacks it into a special